pkg/audit/service.go

```diff
@@ -13,52 +13,62 @@ import (
 	"storj.io/storj/pkg/pointerdb/pdbclient"
 	"storj.io/storj/pkg/provider"
 	"storj.io/storj/pkg/transport"
-	"storj.io/storj/pkg/utils"
 )
 
 // Service helps coordinate Cursor and Verifier to run the audit process continuously
 type Service struct {
 	Cursor   *Cursor
 	Verifier *Verifier
 	Reporter reporter
-	errs     []error
 }
 
 // Config contains configurable values for audit service
 type Config struct {
-	StatDBPort       string                `help:"port to contact statDB client" default:":9090"`
-	MaxRetriesStatDB int                   `help:"max number of times to attempt updating a statdb batch" default:"3"`
-	Pointers         pdbclient.Client      `help:"Pointers for a instantiation of a new service"`
-	Transport        transport.Client      `help:"Transport for a instantiation of a new service"`
-	Overlay          overlay.Client        `help:"Overlay for a instantiation of a new service"`
-	ID               provider.FullIdentity `help:"ID for a instantiation of a new service"`
-	Interval         time.Duration         `help:"how frequently segements should audited" default:"30s"`
+	APIKey           string        `help:"APIKey to access the statdb"`
+	SatelliteAddr    string        `help:"address to contact services on the satellite"`
+	MaxRetriesStatDB int           `help:"max number of times to attempt updating a statdb batch" default:"3"`
+	Interval         time.Duration `help:"how frequently segments are audited" default:"30s"`
 }
 
 // Run runs the repairer with the configured values
 func (c Config) Run(ctx context.Context, server *provider.Provider) (err error) {
-	service, err := NewService(ctx, c.StatDBPort, c.MaxRetriesStatDB, c.Pointers, c.Transport, c.Overlay, c.ID)
+	ca, err := provider.NewCA(ctx, 12, 4)
```

Comment:
I don't think we should be creating identity here, but I think it's fine for now.